django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,87 @@
+from django.core import checks
+from django.db import models
+from django.db.models.expressions import DatabaseDefault
+from django.utils.translation import gettext_lazy as _
+
+__all__ = ("BigSerialField", "SmallSerialField", "SerialField")
+
+
+class SerialFieldMixin:
+    db_returning = True
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("default", DatabaseDefault())
```

Comment:
The default value of `default` is `NOT_PROVIDED`, which is more appropriate, so we don't need this line:

```suggestion
```

Reply:
Maybe we could plug the `DatabaseDefault` in `get_prep_value`?